jobserver/releases.py

```diff
@@ -112,6 +137,20 @@ def create_release(workspace, backend, created_by, requested_files, **kwargs):
         requested_files=requested_files,
         **kwargs,
     )
+
+    if new_style:
+        for f in requested_files:
+            ReleaseFile.objects.create(
+                release=release,
+                workspace=release.workspace,
+                created_by=created_by,
+                name=f["name"],
+                path=f["path"],
```

Comment:
There is no path field in the json, just name, which is effectively the path.

https://github.com/opensafely-core/release-hatch/blob/main/hatch/schema.py#L27

Reply:
The path for a release file seems to be path on the server, so should be NULL here?